src/app/templates/index.ftl

```diff
@@ -32,6 +32,10 @@
                 </a>
             </button>
         </#if>
+        <button id="run-workflow" class="ml-5 btn-link btn btn-lg"><a href="${appUrl}/run-workflow">Run workflow
+                <@customSvg />
+            </a>
+        </button>
```

Comment:
This can be removed if conflicts with other tickets.

Reply:
Yes, see mrc-2326. This button should only be made visible when all other workflows work is complete.